android/sdl_android/src/androidTest/java/com/smartdevicelink/test/TestValues.java

```diff
@@ -427,6 +430,13 @@
     public static final DynamicUpdateCapabilities GENERAL_DYNAMICUPDATECAPABILITIES = new DynamicUpdateCapabilities();
     public static final WindowState GENERAL_WINDOWSTATE = new WindowState();
 
+    public static final SeatOccupancy GENERAL_SEAT_OCCUPANCY = new SeatOccupancy();
+    public static final List<SeatStatus> GENERAL_SEATS_OCCUPIED = new ArrayList<SeatStatus>(1);
+    public static final JSONArray JSON_GENERAL_SEATS_OCCUPIED = new JSONArray();
```

Comment:
@kostyaBoss I would add this declaration `public static final JSONObject JSON_OBJECT_GENERAL_SEATS_BELTED = new JSONObject();` where the other `JSONObjects` are declared (around line 604)